modules/@apostrophecms/doc-type/index.js

```diff
@@ -1463,6 +1463,13 @@ module.exports = {
           }
         },
 
+        noAttachments: {
```

Comment:
Following our pattern I think this should be `attachments` with `def: true`. The conditional later can explicitly look for `!== false` to add the attachments like normal.